silverback/settings.py

```diff
@@ -26,7 +28,8 @@ class Settings(BaseSettings, ManagerAccessMixin):
     APP_NAME: str = "bot"
 
     BROKER_CLASS: str = "taskiq:InMemoryBroker"
-    BROKER_URI: str = ""
+    BROKER_URI: str = ""  # to be deprecated in 0.6
```

Comment:
Can maybe merge it with the kwargs, so that doesn't need separate handling below